tests/init_arg/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use canbench_rs::bench;
+
+// A benchmark that does nothing.
+#[bench]
+fn no_changes_test() {}
+
+#[ic_cdk_macros::init]
+pub fn init(arg: String) {
+    assert_eq!(arg, "hello".to_string());
+}
+
+fn main() {}
```

Comment:
This test is a bit brittle, in the sense that, if I were to delete the `init` method here, the `benchmark_works_with_init_args` would still pass.

I think we can modify this canister slightly to make the test stronger. The advantage with this change is that the results of the benchmark are dependent on the init arguments being passed in, and so if the `init` method here was never invoked, the test will fail.

```suggestion
use canbench_rs::bench;
use std::cell::RefCell;

thread_local! {
    static STATE: RefCell<String> = RefCell::new(String::new());
}

// A benchmark that prints the state.
#[bench]
fn print_state() {
    println!("{}", STATE.with(|s| s.borrow().clone()));
}

#[ic_cdk_macros::init]
pub fn init(arg: String) {
    STATE.with(|s| * s.borrow_mut() = arg);
}

fn main() {}
```